src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -717,12 +717,7 @@ def downloadImage(self, imageDestination):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
-        ## Initialize image library
-        resultImage = Image.new('L', (256, 288), 'white')
-        pixels = resultImage.load()
-
-        ## Y coordinate of current pixel
-        line = 0
+        imageData = []
```

Comment:
Next time, please create your branch from `Development`, not your feature branch to avoid having the changes also here.

Reply:
Yes, my fault! Sry ;)

Reply:
Just merged the mentioned feature branch, but changes still visible in this PR. Can you merge Development in this PR please?